src/libraries/Microsoft.Extensions.FileProviders.Physical/Directory.Build.props

```diff
@@ -2,5 +2,6 @@
   <Import Project="..\Directory.Build.props" />
   <PropertyGroup>
     <IsAspNetCoreApp>true</IsAspNetCoreApp>
+    <UnsupportedOSPlatforms>browser</UnsupportedOSPlatforms>
```

Comment:
How is this unsupported on browser? Is the whole assembly not supported on browser?

Reply:
this project references System.IO.FileSystem.Watcher and that is not supported in browser:
https://github.com/dotnet/runtime/blob/7aaffcb2b3d42c79a23a86aecd6d0e2df049ee44/src/libraries/System.IO.FileSystem.Watcher/Directory.Build.props#L6

Reply:
I see, so then should we add a runtime assembly for Browser that throws PNSE?

cc: @marek-safar @steveisok

Reply:
/cc @lewing

Reply:
> I see, so then should we add a runtime assembly for Browser that throws PNSE?

Yes which is what this setting does

Reply:
This setting doesn't generate a PNSE automatically. In order to generate PNSE we need to add a `-Browser` tfm and then set the flag to generate a pnse when TargetsBrowser == true. So then it seems like we should do that.

Reply:
Thanks for catching this, I've addressed @safern's comment in https://github.com/dotnet/runtime/pull/55664/commits/75fcf967bd423b351caf290e62683296dfafde39, please let me know if something else (related to mark the project as unsupported in Browser) needs to be done.